server/controllers/routes/user/cohort/getSpecificCohort.js

```diff
@@ -0,0 +1,21 @@
+const {
+  cohort: { getSpecificCohort },
+} = require('../../../../database/queries');
+
+exports.getSpecificCohort = async (req, res, next) => {
```

Comment:
@Mu7ammadAbed Since you don't use `user` in the actual api route, it might make more sense to put these files at `server/controllers/routes/cohort` in the repo instead of `server/controllers/routes/user/cohort`.  That might make it easier for other developers to find them in the future (and also means you don't have to write as many `../../..` 😅